doc/api/esm.md

```diff
@@ -1000,19 +1000,15 @@ _isImports_, _conditions_)
 
 **PATTERN\_KEY\_COMPARE**(_keyA_, _keyB_)
 
-> 1. Assert: _keyA_ ends with _"/"_ or contains only a single _"\*"_.
-> 2. Assert: _keyB_ ends with _"/"_ or contains only a single _"\*"_.
-> 3. Let _baseLengthA_ be the index of _"\*"_ in _keyA_ plus one, if _keyA_
->    contains _"\*"_, or the length of _keyA_ otherwise.
-> 4. Let _baseLengthB_ be the index of _"\*"_ in _keyB_ plus one, if _keyB_
->    contains _"\*"_, or the length of _keyB_ otherwise.
+> 1. Assert: _keyA_ contains only a single _"\*"_.
+> 2. Assert: _keyB_ contains only a single _"\*"_.
+> 3. Let _baseLengthA_ be the index of _"\*"_ in _keyA_.
+> 4. Let _baseLengthB_ be the index of _"\*"_ in _keyB_.
```

Comment:
```suggestion
> 3. Let _baseLengthA_ be the index of _"\*"_ in _keyA_ plus one.
> 4. Let _baseLengthB_ be the index of _"\*"_ in _keyB_ plus one.
```

Reply:
Curious what the use case for adding one is, if `baseLengthA` and `baseLengthB` aren't used elsewhere? It made sense before, since the key could also use a trailing `/` instead of `*`, but now they both always contain a `*`.

Reply:
I guess you're right, that "plus one" is irrelevant